Model/Api/ShippingMethods.php

```diff
@@ -136,8 +137,13 @@ class ShippingMethods implements ShippingMethodsInterface
      */
     private $customerFactory;
 
+    /**
+     * @var PriceHelper
+     */
+    private $priceHelper;
+
     // Totals adjustment threshold
-    private $threshold = 0.01;
+    private $threshold = 0.5;
```

Comment:
I am curious about this threshold, why do we update it to 0.5? For me, I think it is bigger than expect. Why not 0.1? is there any principle on such a threshold?

Reply:
Actually the intended purpose for this threshold was dealing with 1+ cent totals mismatch. Since there was nothing here calculated by the plugin but all the values taken from Magento we did not have the mismatch case so far. Anyway, what was eventually rounded / cut off from shipping amount was added to the tax amount to re-balance the total. Now that eventual discount is subtracted I started getting bugsnag logs and realized that this threshold specifies cents, so it was too low.

Reply:
I don't believe that number makes sense because you are comparing what theoretically should be the difference between two integers.  I understand that it may have initially been comparing two floats.

The logical value for this in the context of a check like
```
if (abs($diff) >= $this->threshold) { ...
```
would be `private $threshold = 1;`  Yes, 0.5 would accomplish a similar thing, but the integer 1 is more clear.

Reply:
@mccottry Correct. Fixed.